src/manage-candidate.php

```diff
@@ -0,0 +1,450 @@
+<?php
+
+include_once str_replace('/', DIRECTORY_SEPARATOR, __DIR__ . '/include/classes/File-utils.php');
+require_once FileUtils::normalizeFilePath(__DIR__ . '/includes/classes/db-connector.php');
+require_once FileUtils::normalizeFilePath(__DIR__ . '/includes/session-handler.php');
+require_once FileUtils::normalizeFilePath(__DIR__ . '/includes/classes/session-manager.php');
+require_once FileUtils::normalizeFilePath(__DIR__ . '/includes/classes/query-handler.php');
```

Comment:
This is incorrect.

![image](https://github.com/BSIT-3-1-APPDEV/PUPSRC-AutomatedElectionSystem/assets/162773681/0425168e-e2c4-43df-8aae-c5084037678c)